cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -208,7 +217,8 @@ func (i *deployCommand) Exec(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	if installed, err := shared.SilentlyInstallOperatorIfNotExists(i.flags.Project, "", i.Client); err != nil {
+	installationChannel := shared.KogitoChannelType(i.flags.Channel)
```

Comment:
Please avoid temps like that, just use `shared.KogitoChannelType(i.flags.Channel)` in the function call or create a handler in `OperatorFlags` like `getKogitoChannelType`, so we avoid implicit conversion when using the flag.

Reply:
done